### ISSUE TITLE
Ctrl-s to open horizontal split stopped working (tested in :Niffler and :NifflerMRU)

### ISSUE THREAD

**Issue by marlun** [pgdouyon/vim-niffler#29]
Not sure if another plugin has updated and started messing with the shortcut or if there's a bug in Niffler. If I run `:verbose imap <c-s>` I get:

```
i  <C-S>         <Plug>Isurround
    Last set from ~/Sources/dotfiles/vim/bundle/vim-surround/plugin/surround.vim
```

but I don't know if that affects the Niffler window?

**Comment by pgdouyon** [comment]
Hmmm that's interesting, I can't reproduce it in Neovim, but I can reproduce it in Vim.  I suppose that was one of the many improvements/bugs Neovim fixed.  You can easily reproduce the problem since `<C-S>` isn't implemented with maps but with `getchar()`.  In Vim at the command prompt type `:echo getchar()` and then you type `<C-S>` until your fingers bleed but it seems `getchar()` will never return that value.  It will return other chorded combinations just fine.

I'm guessing Vim throws away `<C-S>` because many terminals use it to implement Flow Control (http://unix.stackexchange.com/questions/72086/ctrl-s-hang-terminal-emulator), but that still seems kind of silly.  I could come up with a backup mapping in case `<C-S>` doesn't work, any suggestions for one?

**Comment by marlun** [comment]
I was almost sure it had been working for me in vim before but I guess it's just that I used neovim before. I've switched back to vim again for the time being though.

I could fix it by disabling sending START/STOP signals in the terminal using this:

```
stty -ixon -ixoff
```

I never use that functionality anyway.

Not sure what should be used as a backup mapping though. Maybe we can just add a FAQ to the docs and add a "Ctrl-s for opening in a vertical split doesn't work for me" question with the solution above as an alternative? :smile:
